src/staging/validation.py

```diff
@@ -26,17 +26,20 @@ def load_schema(file_path: str = "./config/contributors_schema.toml") -> dict:
 
     # Check if Data_Schema.toml exists
     if file_exists:
-        # Load toml data schema into dictionary if toml file exists
-        toml_dict = toml.load(file_path)
+        try:
+            # Open the file and load toml data schema into dictionary
+            with open(file_path, 'rb') as file:
+                toml_dict = tomli.load(file)
+            return toml_dict
+        except tomli.TOMLDecodeError as e:
+            ValidationLogger.error(f"Failed to decode TOML file: {e}")
+            return None
     else:
-        # Return False if file does not exist
+        # Return None if file does not exist
         ValidationLogger.warning(
             "Validation schema does not exist! Path may be incorrect"
         )
-        return file_exists
-
-    return toml_dict
-
+        return None
```

Comment:
Forgive my ignorance, but the function previously returned `toml_dict`, but in these changes the function returns `None` in both scenarios. Is this correct?

Reply:
@Ryan2Y79 no, look at line 33, we return toml_dict there